Code/GraphMol/Wrap/Atom.cpp

```diff
@@ -104,14 +104,24 @@ std::string AtomGetSmarts(const Atom *atom, bool doKekule, bool allHsExplicit,
 }
 
 void SetAtomMonomerInfo(Atom *atom, const AtomMonomerInfo *info) {
-  atom->setMonomerInfo(info->copy());
+  if(!info) {
+    atom->setMonomerInfo(nullptr);
+  } else {
+    atom->setMonomerInfo(info->copy());
+  }
 }
```

Comment:
@ptosco would do this with a ternary operator, but I think it's fine like this.
;-)

Reply:
Since the crash was caused by the python wrapper I’m not sure what a c++ check would accomplish.

the c++ change is a mem leak fix which is also hard to test.

Reply:
As for the ternary, I was going to throw an exception but then I fixed the memory leak and thought better of it